diesel/src/macros/mod.rs

```diff
@@ -198,6 +198,23 @@ macro_rules! __diesel_column {
 /// # fn main() {}
 /// ```
 ///
+/// If you have a column with the same name as a rust reserved keyword, you can use
```

Comment:
Nit: Is the name of the programming language we use written as proper name with a capital letter at the start or is lowercase okay? This looks wrong.